libs/core/tests/serializers/signed_integers.cpp

```diff
@@ -32,7 +32,7 @@ namespace fetch {
 
 namespace serializers {
 
-TEST(MsgPacker, signed_integers)
+TEST(MsgPacker, signed_integers)  // NOLINT
```

Comment:
Overly long function - ideally this will get split up into smaller tests, but I think it's important to enable the check first (same reasoning as for explicit ctors)